pyigm/metallicity/mcmc.py

```diff
@@ -297,9 +297,16 @@ def loadmodel(self, data, model):
         """
         
         #load the model in a format that can be handled later on
-        fil=open(model,'br')
-        modl=pickle.load(fil)
-        fil.close()
+        try:
```

Comment:
Most of the lines in the try/except 
are repeated.   Please pull those out, i.e.

```
modl=pickle.load(fil)
fil.close()
```